cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -73,7 +73,7 @@ The index name must be unique among both indexes and constraints.
 
 The command is optionally idempotent, with the default behavior to throw an error if you attempt to create the same index twice.
 With `IF NOT EXISTS`, no error is thrown and nothing happens should an index with the same name, schema or both already exist.
```

Comment:
```suggestion
With `IF NOT EXISTS`, no error is thrown and nothing happens if an index with the same name, schema or both already exist.
```

Reply:
```suggestion
With `IF NOT EXISTS`, no error is thrown and nothing happens if an index with the same name, schema or both already exists.
```
to be consistent with the change on constraints

Reply:
Yes, you're right, it should be _exists_ as it refers to a single entity.